pkg/local/capacity.go

```diff
@@ -158,8 +158,9 @@ func getLoopDeviceCapacity() []*StorageCapacity {
 		}
 		percent := 0.9
 		if types.GlobalConfigVar.LocalSparseTotalAvailablePercent != "" {
-			p, _ := strconv.Atoi(types.GlobalConfigVar.LocalSparseTotalAvailablePercent)
-			percent = float64(p)
+			if p, err := strconv.ParseFloat(types.GlobalConfigVar.LocalSparseTotalAvailablePercent, 32); err == nil && p < 1 {
```

Comment:
Since this variable is named "Percent", I would not expect it to be less than 1.

Also, for sparse files, over 100% should also be acceptable?

Reply:
It's just the space available in a directory, not the sparse-file specific.

Reply:
"percent" means 1/100, which is different from implementation. Should we fix this? Either change the semantic or the name of this env to avoid confusions.